patches/minecraft/net/minecraft/server/network/ServerGamePacketListenerImpl.java.patch

```diff
@@ -87,11 +87,16 @@
 +                  if(!ServerGamePacketListenerImpl.this.f_9743_.canInteractWith(entity, 1.5D)) return; //Forge: If the entity cannot be reached, do nothing. Original check was dist < 6, range is 4.5, so vanilla used padding=1.5
                    ItemStack itemstack = ServerGamePacketListenerImpl.this.f_9743_.m_21120_(p_143679_).m_41777_();
                    InteractionResult interactionresult = p_143680_.m_143694_(ServerGamePacketListenerImpl.this.f_9743_, entity, p_143679_);
-+                  if (net.minecraftforge.common.ForgeHooks.onInteractEntityAt(f_9743_, entity, entity.m_20182_(), p_143679_) != null) return;
                    if (interactionresult.m_19077_()) {
-                      CriteriaTriggers.f_10565_.m_61494_(ServerGamePacketListenerImpl.this.f_9743_, itemstack, entity);
-                      if (interactionresult.m_19080_()) {
-@@ -1558,6 +_,8 @@
+@@ -1552,12 +_,15 @@
+ 
+                public void m_142143_(InteractionHand p_143682_, Vec3 p_143683_) {
+                   this.m_143678_(p_143682_, (p_143686_, p_143687_, p_143688_) -> {
+-                     return p_143687_.m_7111_(p_143686_, p_143683_, p_143688_);
++                     InteractionResult onInteractEntityAtResult = net.minecraftforge.common.ForgeHooks.onInteractEntityAt(f_9743_, entity, p_143683_, p_143682_);
++                     return onInteractEntityAtResult != null ? onInteractEntityAtResult : p_143686_.m_7111_(p_143686_, p_143683_, p_143688_);
```

Comment:
Again...the if statement can be condensed down to one line. This makes three line changes (2 additions 1 removal) while the other only causes two (2 additions).

Reply:
I see what you mean now.
I don't understand why I was so dense as to seeing that change.